packages/ui-progress/src/ProgressBar/theme.js

```diff
@@ -22,77 +22,89 @@
  * SOFTWARE.
  */
 
-export default function generator({ borders, colors, spacing, typography }) {
-  return {
-    color: colors.textDarkest,
+/**
+ * Generates the theme object for the component from the theme and provided additional information
+ * @param  {Object} theme The actual theme object.
+ * @param  {Object} themeOverride User provided overrides of the default theme mapping.
+ * @return {Object} The final theme object with the overrides and component variables
+ */
+const generateComponentTheme = (theme, themeOverride = {}) => {
+  const { borders, colors, spacing, typography, key: themeName } = theme
+
+  const themeSpecificStyle = {
+    canvas: {
+      color: theme['ic-brand-font-color-dark'],
+      meterColorBrand: theme['ic-brand-primary']
+    },
+    'canvas-high-contrast': {
+      meterBorderColorInverse: colors?.borderLightest,
+      meterColorBrandInverse: colors?.backgroundLightest,
+      meterColorSuccessInverse: colors?.backgroundLightest,
+      meterColorInfoInverse: colors?.backgroundLightest,
+      meterColorAlertInverse: colors?.backgroundLightest,
+      meterColorWarningInverse: colors?.backgroundLightest,
+      meterColorDangerInverse: colors?.backgroundLightest
+    }
+  }
+  themeSpecificStyle['canvas-a11y'] = themeSpecificStyle['canvas-high-contrast']
```

Comment:
Is this needed? As I see we don't have a 'canvas-a11y' theme, I guess its an older name for the 'canvas-high-contrast' theme

Reply:
Thanks for pointing this out, I removed it (and from any other usages in next too)!